app/kumactl/cmd/install/install_transparent_proxy.go

```diff
@@ -123,10 +121,14 @@ runuser -u kuma-dp -- \
 				return errors.Errorf("transparent proxy will work only on Linux OSes")
 			}
 
-			if args.ModifyIptables && args.User == "" && args.UID == "" {
+			if args.User == "" && args.UID == "" {
 				return errors.Errorf("--kuma-dp-user or --kuma-dp-uid should be supplied")
 			}
 
+			if args.RedirectAllDNSTraffic && args.RedirectDNS {
+				return errors.Errorf("one of --redirect-dns or --redirect-all-dns-traffic should be specified")
```

Comment:
nit: I'd phrase this as
```suggestion
return errors.Errorf("exactly one of --redirect-dns or --redirect-all-dns-traffic should be specified")
```

Reply:
None of them is also a valid case, I think I probably should do `only one of ...`?